ci/snaps/snaps_os_tmplt.yaml

```diff
@@ -65,7 +65,7 @@ openstack:
   images:
     - image:
         os_creds_name: admin-creds
-        name: snaps-os-image-{{ build_id }}
+        name: ubuntu-16-04
```

Comment:
Should make this configurable but this is fine. Also, we can also have a default file and URL added for images too but this should be fine for the time being; however, it requires manual (or some other method) uploading of that specific image which could cause headaches to some users.